src/searcher.rs

```diff
@@ -26,23 +25,10 @@ pub struct DoubleArrayTrieSearcher<'a, 'b> {
     start_pos: usize,
 }
 
-impl<'a, 'b> DoubleArrayTrieSearcher<'a, 'b> {
-    pub fn search_step_to_str(&self, step: &SearchStep) -> String {
```

Comment:
Removed this method from public API since it feels like it was only used in unit test.